### ISSUE TITLE
Limiting webcam detection area

### ISSUE THREAD

**Issue by luppahannu** [opentrack/opentrack#1125]
For tracker inputs that use a webcam, could there be a way to crop the area from the webcam picture, so the tracker would only search the dots/markers from a specific area of the image? I'm assuming searching for the markers would be faster when the area is smaller.

**Comment by sthalik** [comment]
> I'm assuming searching for the markers would be faster when the area is smaller.

Not really. The tracker only searches a small area around the previous detection on a subsequent frame, additionally with a similar size.
